woocommerce/woocommerce-admin
# Onboarding: Add the task list dashboard

Fixes #1896 

Adds the base for the task list dashboard.

A couple design question /cc @jameskoster 

* I attempted to replace with Gridicons since we already have those imported.  Not all of the icons match up well, so if you want to change any or think we should just individually import the ones in Figma, we can do so.
* Should the header bar be fixed to the top of the screen?  Currently it's fixed on desktop and sits statically on mobile.

### Screenshots
<img width="1378" alt="Screen Shot 2019-07-15 at 6 47 21 PM" src="https://user-images.githubusercontent.com/10561050/61211096-28210700-a731-11e9-9589-78bb56632c55.png">

### Detailed test instructions:

1. Make sure you have the onboarding feature flag enabled and have completed or skipped the profiler.
2. Change `requiredTasksComplete` to `false` in dashboard/index.js.
3. Visit the dashboard.
4. Check that the dashboard and task list match those in Figma.
